### ISSUE TITLE
Quill css doesn't apply offline

### ISSUE THREAD

**Issue by JCMilito** [quilljs/quill#2962]
Please describe the a concise description and fill out the details below. It will help others efficiently understand your request and get to an answer instead of repeated back and forth. Providing a [minimal, complete and verifiable example](https://stackoverflow.com/help/mcve) will further increase your chances that someone can help.

**Steps for Reproduction**

1. Visit [quilljs.com, jsfiddle.net, codepen.io]
2. Step Two
3. Step Three

**Expected behavior**:

**Actual behavior**:

**Platforms**:

Include browser, operating system and respective versions

**Version**:

Run `Quill.version` to find out

**Comment by JCMilito** [comment]
Hi,

So, I'm developing an Ionic app using Quill.
When I include <link rel="stylesheet" href="//cdn.quilljs.com/1.3.7/quill.snow.css"> in my HTML, everything works fine.
Turns out that, when I try to apply the style from a local file, it doesn't work properly. 
I even copied the contents of the file located in cdn.quilljs.com/1.3.7/quill.snow.css in my page.scss, to no avail. All I got is a giant triangle where the editor should be.
What I'm doing wrong? 

Thx in advance

**Comment by JCMilito** [comment]
Hi,

So, I'm developing an Ionic app using Quill.
When I include in my HTML, everything works fine.
Turns out that, when I try to apply the style from a local file, it doesn't work properly.
I even copied the contents of the file located in cdn.quilljs.com/1.3.7/quill.snow.css in my page.scss, to no avail. All I got is a giant triangle where the editor should be.
What I'm doing wrong?

Thx in advance
